src/main/resources/db/80/V1000_57__maimai_circle_event.sql

```diff
@@ -0,0 +1,243 @@
+(250822061, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250822091, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250822092, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250905011, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250905021, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250905041, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918011, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918012, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918013, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918014, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918015, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918016, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918017, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918018, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918019, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918021, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918022, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918023, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918024, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918025, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918026, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918027, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918028, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918031, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918041, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918042, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918043, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918051, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918061, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918062, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918071, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250918111, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250926011, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250926021, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250926022, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
+(250926121, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '1'),
```

Comment:
```suggestion
(250926121, '2029-01-01 00:00:00.000000', '2019-01-01 00:00:00.000000', 0, '0'),
```
这个不建议开

另外，确定后面写 '0' 或者 '1' 是能识别的吗，我这里是写 true false 的

Reply:
MariaDB会尝试将字符串`'1'`转换为数字`1`<sup>其实应该是写脚本的时候漏了个b</sup>，文档[13.1.1 Numeric Data Type Syntax](https://dev.mysql.com/doc/refman/8.0/en/numeric-type-syntax.html)中讲述TINYINT和BOOLEAN数据类型在本质上是完全相同，并且经过测试是可以正常发放event，因此问题不大

> BOOL, BOOLEA
> These types are synonyms for TINYINT(1). A value of zero is considered false. Nonzero values are considered true: